docs/content/en/docs/usage/observability/opentelemetry.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Tracing with OpenTelemetry"
+weight: 11
+---
+
+## Tracing in Fission
+
+Currently, Fission supports outputting traces to an OpenTracing Jaeger-formatted trace collection endpoint. This is great as it provides lots of insight into what Fission is doing and helps operators understand and maintain.
+
+## OpenTelemetry
+
+OpenTelemetry is a set of APIs, SDKs, tooling and integrations that are designed for the creation and management of telemetry data such as traces, metrics, and logs. The project provides a vendor-agnostic implementation that can be configured to send telemetry data to the backend(s) of your choice. It supports a variety of popular open-source projects including Jaeger and Prometheus.
+
+## Setting up
+
+### Prerequisite
+
+- Docker
+- Kubernetes cluster (the document uses a `kind` cluster)
+- [Helm](https://helm.sh/) (This post assumes helm 3 in use)
+- kubectl and kubeconfig configured
+
+### Setup OTEL Collector and Jaeger
+
+We will be using the definition files available in the [OpenTelemetry-Go](https://github.com/open-telemetry/opentelemetry-go.git) repository for an easy setup.
+Follow the steps below:
+
+```sh
+# clone the repository
+git clone https://github.com/open-telemetry/opentelemetry-go.git
```

Comment:
The process seems a bit hectic of cloning repos etc. also not consistent across version of opentelemetry.

https://github.com/open-telemetry/opentelemetry-operator
Should we use an opentelemetry operator instead?

We can include it Fission helm chart as well going  ahead once it stabilizess.